schema/install_schema.go

```diff
@@ -7,65 +7,65 @@ import (
+	FS   string `json:"fs,omitempty" mapstrcuture:"fs" yaml:"fs,omitempty"`
 }
 
 type ElementalPartitions struct {
-	OEM        *Partition `json:"oem,omitempty" mapstructure:"oem"`
-	Recovery   *Partition `json:"recovery,omitempty" mapstructure:"recovery"`
-	State      *Partition `json:"state,omitempty" mapstructure:"state"`
-	Persistent *Partition `json:"persistent,omitempty" mapstructure:"persistent"`
+	OEM        *Partition `json:"oem,omitempty" mapstructure:"oem" yaml:"oem,omitempty"`
+	Recovery   *Partition `json:"recovery,omitempty" mapstructure:"recovery" yaml:"recovery,omitempty"`
+	State      *Partition `json:"state,omitempty" mapstructure:"state" yaml:"state,omitempty"`
+	Persistent *Partition `json:"persistent,omitempty" mapstructure:"persistent" yaml:"persistent,omitempty"`
 }
 
 // BundleSchema represents the bundle block which can be used in different places of the Kairos configuration. It is used to reference a bundle and its confguration.
 type BundleSchema struct {
-	DB         string   `json:"db_path,omitempty"`
-	LocalFile  bool     `json:"local_file,omitempty"`
-	Repository string   `json:"repository,omitempty"`
-	Rootfs     string   `json:"rootfs_path,omitempty"`
-	Targets    []string `json:"targets,omitempty"`
+	DB         string   `json:"db_path,omitempty" yaml:"db_path,omitempty"`
+	LocalFile  bool     `json:"local_file,omitempty" yaml:"local_file,omitempty"`
+	Repository string   `json:"repository,omitempty" yaml:"repository,omitempty"`
+	Rootfs     string   `json:"rootfs_path,omitempty" yaml:"rootfs_path,omitempty"`
+	Targets    []string `json:"targets,omitempty" yaml:"targets,omitempty"`
 }
 
 // GrubOptionsSchema represents the grub options block which can be used in different places of the Kairos configuration. It is used to configure grub.
 type GrubOptionsSchema struct {
-	DefaultFallback      string `json:"default_fallback,omitempty" description:"Sets default fallback logic"`
-	DefaultMenuEntry     string `json:"default_menu_entry,omitempty" description:"Change GRUB menu entry"`
-	ExtraActiveCmdline   string `json:"extra_active_cmdline,omitempty" description:"Additional Kernel option cmdline to apply just for active"`
-	ExtraCmdline         string `json:"extra_cmdline,omitempty" description:"Additional Kernel option cmdline to apply"`
-	ExtraPassiveCmdline  string `json:"extra_passive_cmdline,omitempty" description:"Additional Kernel option cmdline to apply just for passive"`
-	ExtraRecoveryCmdline string `json:"extra_recovery_cmdline,omitempty" description:"Set additional boot commands when booting into recovery"`
-	NextEntry            string `json:"next_entry,omitempty" description:"Set the next reboot entry."`
-	SavedEntry           string `json:"saved_entry,omitempty" description:"Set the default boot entry."`
+	DefaultFallback      string `json:"default_fallback,omitempty" description:"Sets default fallback logic" yaml:"default_fallback,omitempty"`
```

Comment:
I think grub options is an open map[string]string, so this would miss any extra configurations that the user might have

Reply:
@nianyush Yes, GrubOptionsSchema is a map in Install Struct, we need to handle that if we switch from Install to Install Schema

cc: @santhoshdaivajna